src/geo/map/legends/custom-legend-model.js

```diff
@@ -7,6 +7,24 @@ var CustomLegendModel = StaticLegendModelBase.extend({
       type: 'custom',
       items: []
     });
+  },
+
+  parse: function (r, opts) {
```

Comment:
Why are we using `parse` and not doing this in the constructor?

From Backbone's docs:
```
parse is called whenever a model's data is returned by the server,
```

The problem I see with this approach is that then we have to use `parse:true` everywhere, which is something easy to forget, don't you think?